src/components/SelectOption/SelectOption.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import usersFromServer from '../../api/users';
```

Comment:
[CODE STYLE] - Don't use one-letter variable naming. While 'usersFromServer' is not a one-letter variable, it could be improved for clarity. Consider a more descriptive name like 'availableUsers' or 'userOptions'.